src/main/java/com/artipie/Pie.java

```diff
@@ -117,6 +118,9 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
                     case "gem":
                         slice = CompletableFuture.completedStage(new GemSlice(storage));
                         break;
+                    case "rpm":
```

Comment:
@g4s8 this switch/case is pretty ugly and error prone. How about we extract a class `SliceFromCfg` (it can be protected) which will contain a map of slices by their type? Something like `Map<String, Function<RepoConfig, CompletitionStage<Slice>>`. It can be tested separately and further updates are done only on the `map`. WDYT? It can be even made extendable so that a user can put its own slice definitions but this is maybe an overkill.

Reply:
@Vatavuk thanks, fixed